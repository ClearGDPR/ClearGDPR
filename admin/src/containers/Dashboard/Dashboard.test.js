import React from 'react';
import DashboardContainer from './Dashboard';
import { shallow } from 'enzyme';
import session from 'helpers/Session';

jest.mock('helpers/Session');

beforeEach(() => {
  session.getToken.mockReturnValue('token');
});

describe('(Container) Dashboard', () => {
  it('Should render without an error', () => {
    const component = shallow(<DashboardContainer />);
    expect(component).toMatchSnapshot();
  });
});
