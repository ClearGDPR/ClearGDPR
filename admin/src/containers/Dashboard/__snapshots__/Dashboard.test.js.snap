// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Container) Dashboard Should render without an error 1`] = `
<section
  className="cards"
>
  <SubjectsListContainer />
  <EventsListContainer />
  <div
    className="row"
  >
    <[object Object] />
  </div>
</section>
`;
